x/interchainstaking/keeper/keeper.go

```diff
@@ -634,3 +639,7 @@ func DetermineAllocationsForRebalancing(currentAllocations map[string]math.Int,
 
 	return out
 }
+
```

Comment:
If we are adding setEpochsKeeper, do we need to remove epochkeeper from NewKeeper (I added in v1.2.5)